core/src/trezor/ui/layouts/tt/__init__.py

```diff
@@ -6,7 +6,12 @@
 from trezor.messages import ButtonAck, ButtonRequest
 from trezor.wire import ActionCancelled, context
 
-from ..common import button_request, interact
+from ..common import (
+    button_request,
+    get_last_transition_out,
+    interact,
+    set_last_transition_out,
+)
```

Comment:
no need for those here and in `tr`

Reply:
removed https://github.com/trezor/trezor-firmware/commit/07bd5dda9eb2378e293a6e5ed216beb3c2429bbc